megalinter/MegaLinter.py

```diff
@@ -111,11 +124,31 @@ def __init__(self, params=None):
         self.has_git_extraheader = False
         self.has_updated_sources = 0
         self.fail_if_updated_sources = (
-            config.get("FAIL_IF_UPDATED_SOURCES", "false") == "true"
+            config.get(self.request_id, "FAIL_IF_UPDATED_SOURCES", "false") == "true"
         )
         self.flavor_suggestions = None
+
         # Initialize plugins
-        plugin_factory.initialize_plugins()
+        plugin_factory.initialize_plugins(self.request_id)
+
+        # Copy node_modules in current folder if necessary
+        if (
+            os.path.isdir("/node_deps")
+            and len(os.listdir("/node_deps")) > 0
+            and pre_post_factory.has_npm_or_yarn_commands(self.request_id)
+        ):
+            workspace_node_modules = os.path.join(self.workspace, "node_modules")
+            copytree("/node_deps", workspace_node_modules)
```

Comment:
This will raise a `FileExistsError` in the common case where the workspace already contains `node_modules`.